roles/unblock-ports/tasks/main.yml

```diff
@@ -13,4 +13,5 @@
   - 587
   - 993
   - 995
+  - 1194
   - 4190
```

Comment:
You'll need another section here.
```yaml
- name: unblock the needed ports for UDP
   ufw: rule=allow port={{ item }} proto=udp
   with_items:
    - 53
    - 1194
```

Reply:
ok wonder what i can tackle next :) ideas ??

Reply:
Well, with Wordpress and plain nginx, vpn, dns, and opencart support, I think this is a complete hosting solution now.